Formula/unbound.rb

```diff
@@ -13,11 +13,13 @@ class Unbound < Formula
 
   depends_on "openssl"
   depends_on "libevent"
+  depends_on "expat"
```

Comment:
1.5.9 without the expat dependency builds when you do have CLT installed.
Sorry but I could not replicate and test your patch as I've already
installed CLT.

On Mon, Jul 18, 2016 at 2:00 PM Frederico Neves <fneves@gmail.com> wrote:

> If this works for you apparently both versions works correctly. I mean the
> ones build with the system and brew versions of expat.
>
> On Mon, Jul 18, 2016 at 1:57 PM Dominyk Tiller <notifications@github.com>
> wrote:
>
>> In Formula/unbound.rb
>> <https://github.com/Homebrew/homebrew-core/pull/3080#discussion_r71185463>
>> :
>>
>> > @@ -13,11 +13,13 @@ class Unbound < Formula
>> >
>> >    depends_on "openssl"
>> >    depends_on "libevent"
>> > +  depends_on "expat"
>>
>> I think if we adopt the code block I proposed that'll get it working for
>> everyone without making 'em install the CLT to get there.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Homebrew/homebrew-core/pull/3080/files/bae0c807e72cb3fde2dff0a9fabb6c40486b6982#r71185463>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/ATkHVWsK9cMmPfsWYG6_AvEt3Xrmjfjuks5qW7BsgaJpZM4JOknY>
>> .
>>
>